staging/src/k8s.io/dynamic-resource-allocation/deviceattribute/pci_linux_test.go

```diff
@@ -0,0 +1,156 @@
+/*
+Copyright 2025 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package deviceattribute
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"reflect"
+	"strings"
+	"testing"
+
+	"k8s.io/utils/ptr"
+
+	resourceapi "k8s.io/api/resource/v1beta2"
+)
+
+func TestGetPCIeRootBAttributeyPCIBusID(t *testing.T) {
+	pciBusID := "0000:01:02.3"
+	pcieRoot := "pci0000:01"
```

Comment:
cc @everpeace 
Everything looks good, but how could the PCIe root of "0000:01:02.3" be "pci0000:01"?
It should be "pci0000:00".